src/components/Modals/WalletModal.tsx

```diff
@@ -1,8 +1,13 @@
 import React, { useEffect } from "react";
-import { useAccount, useConnect } from "wagmi";
+import { useAccount, useConnect, useNetwork, useSwitchNetwork } from "wagmi";
 import { MetaMaskIcon, WalletConnectIcon } from "../../icons";
 import { BaseModal } from ".";
 
+const DEFAULT_NETWORK = {
+  name: "Goerli",
+  id: 5
```

Comment:
I think we should have the network config provided through the /config endpoint to prevent us getting out of sync with the BE when we switch to mainnet

Reply:
that's not currently provided by the `/config` endpoint and should be included as part of the backend refactor so that it we can also support multiple networks -- i believe @pondpiu's next pr includes a network addresses file that will address this